core/modes/modes_not_dragon.talon

```diff
@@ -0,0 +1,32 @@
+mode: sleep
+mode: command
+mode: dictation
+not speech.engine: dragon
+-
+^talon wake [<phrase>]$: speech.enable()
+
+# We define this *only* if the speech engine isn't Dragon, because if you're using Dragon,
+# "wake up" is used to specifically control Dragon, and not affect Talon.
+#
+# It's a useful and well known command, though, so if you're using any other speech
+# engine, this controls Talon.
+^(wake up)+$: speech.enable()
+
+# We define this *only* if the speech engine isn't Dragon, because if you're using Dragon,
+# "go to sleep" is used to specifically control Dragon, and not affect Talon.
+#
+# It's a useful and well known command, though, so if you're using any other speech
+# engine, this controls Talon.
+#
+# For a note about the optional <phrase>, see to_sleep_mode.talon.
```

Comment:
`to_sleep_mode.talon` no longer exists. I wonder if we should move those comments here?